geoapps/utils/utils.py

```diff
@@ -41,6 +41,53 @@
 from sklearn.neighbors import KernelDensity
 
 
+def string_2_numeric(text: str) -> int | float | str:
+    """Converts numeric string representation to int or string if possible."""
+    try:
+        text_as_float = float(text)
+        text_as_int = int(text_as_float)
+        return text_as_int if text_as_int == text_as_float else text_as_float
+    except ValueError:
+        return text
+
+
+def sorted_alphanumeric_list(l: list[str]) -> list[str]:
+    """
```

Comment:
Single letter for variables isn't pep8 compliant.

Reply:
ok